docs/blog/2023-06-27-jupyterlab-myst.md

```diff
@@ -0,0 +1,117 @@
+# Using MyST Markdown in JupyterLab
+
+```{post} 2023-06-26
+:author: Rowan Cockett, Angus Hollands, Steve Purves
+:image: 1
+:excerpt: 2
+```
+
+:::{admonition} Upcoming MyST Webinar
+:class: important
+
+Join the upcoming webinar (5th July 2023 @ 3pm UTC) and get started using MyST in JupyterLab and how it connects to other scientific writing workflows. The webinar is hosted by Rowan & Steve from [Curvenote](https://curvenote.com), and will be an hour walk-through of using [MyST in JupyterLab](https://github.com/executablebooks/jupyterlab-myst).
+
+[Sign up for the webinar](https://www.eventbrite.ca/e/scientific-writing-in-jupyterlab-with-myst-markdown-tickets-666670598707) or watch the recording on YouTube after.
+:::
+
+If you are using JupyterBook or MyST Markdown, you can now see your content directly in JupyterLab with the our new extension [`jupyterlab-myst`](https://github.com/executablebooks/jupyterlab-myst). In this post we will showcase `jupyterlab-myst` to enable rich parsing and rendering of MyST natively in JupyterLab, including:
+
+1. rich authoring components like figures, hover cross-references, tabbed content, and admonitions;
+2. including variables, interactive widgets, and graphs directly inside of Markdown cells in the notebook; and
+3. capturing and displaying rich frontmatter.
+
+```{figure} ./images/jupyterlab-myst.png
+:align: center
+:width: 100%
+
+MyST extension in JupyterLab with a rich authoring experience, frontmatter and inline expressions.
+```
+
+To follow along with the tutorial you can install the plugin using pip and see the full [getting started guide](https://mystmd.org/guide/quickstart-jupyter-lab-myst):
+
+```shell
+pip install jupyterlab_myst
+```
+
+The plugin works best in the latest version of JupyterLab (version 4.0 or greater). We released v2 of our plugin on [June 23rd, 2023](https://github.com/executablebooks/jupyterlab-myst/releases/tag/v2.0.0), so if you have already installed the plugin, you can update to v2!
```

Comment:
is this saying that v2 is compatible with jlab 4.0 and 3.x?

Reply:
We will make v2 compatible with JupyterLab-3. We need to update our dependencies in `package.json` to reflect the compatibility, and I need to run through and check what's broken on 3.0 :P

If we can hold off on 3.0 advertisement, I woudl

Reply:
The plugin does work in 3x series, with the exception of inline ipywidgets. Simple inline execution does work, all of the parsing does work as well!

My plan today is to do a pass on docs, and I think over there we can be a bit clearer about the exact requirements and leave this side open? (e.g. it works "best" with v4..!).